jekyll/_includes/snippets/runner/install-with-web-app-steps.adoc

```diff
@@ -0,0 +1,46 @@
+//== CircleCI web app installation: pre-steps
+In order to install self-hosted runners, you will need to create a namespace and resource class token. Please note that to create resource classes and tokens, you need to be an organization admin in the VCS provider. You can read about namespaces and resource classes on the <<runner-concepts#namespaces-and-resource-classes,Concepts>> page.
+
+You can view your runners on the inventory page, by clicking *Self-Hosted Runners* on the left navigation.
+
+. On the https://app.circleci.com/[CircleCI web app], navigate to *Self-Hosted Runners*.
++
+image::{{site.baseurl}}/assets/img/docs/runnerui_step_one.png[Runner set up, step one - Get started]
+
+. If this is your organization's first time using self-hosted runners, you'll need to create a namespace as the first part of this step. *Each organization can only create a single namespace*. While not required, we suggest using a lowercase representation of your CircleCI account name. CircleCI will populate your org name as the suggested namespace by default in the UI.
+
+. If your organization already use orbs, do not create a new namespace, enter the namespace your organization uses for orbs.
```

Comment:
I don't think this needs to be a separate step, it can be part of the previous step?